app/Resources/views/members/members.html.twig

```diff
@@ -21,10 +21,10 @@
             <table class="table table-hover table-sm table-responsive-md borderless">
                 <thead>
                 <tr>
-                    <th width="15%"> Dag</th>
-                    <th width="30%"> Navn</th>
-                    <th width="15%"> Tlf</th>
-                    <th width="40%"> E-post</th>
+                    <th class="w-15"> Dag</th>
+                    <th class="w-30"> Navn</th>
+                    <th class="w-15"> Tlf</th>
+                    <th class="w-40"> E-post</th>
```

Comment:
Jeg tror ikke det finnes noen css-klasser for `w-15`, `w-30` og `w-40`. Alternativt kan man bruke `style="width: 15%"` osv.

Uansett - er denne templaten egentlig i bruk?
Hvis ikke kan vi heller slette den.

Reply:
Slettet den, ettersom den ikke er i bruk.